### PR TITLE
Added information about accessing host directory on Windows.

### PR DESCRIPTION
**Pull request by jaycode**

### CODE REVIEW

File: docs/sources/userguide/dockerimages.md [hunk 1]
```diff
@@ -242,6 +242,9 @@ Let's create a directory and a `Dockerfile` first.
     $ cd sinatra
     $ touch Dockerfile
 
+If you are using boot2docker on Windows, you may access your host
```
Comment (moxiegirl) [inline docs/sources/userguide/dockerimages.md]: Either Boot2Docker of `boot2docker`.  

by `cd` to `/c/Users/your_user_name`

**Comment by tiborvass** [comment]
@jaycode let us know if you're not available we can carry the PR

**Comment by jaycode** [comment]
Please carry on with the PR for me, thank you,

Jay

**Comment by tiborvass** [comment]
Thanks for your contribution @jaycode ! I carried it at #11997

**Pull request closed without merging**